composer.json

```
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "cakephp/console": "^4.0",
-        "rector/rector": "~0.10.20"
+        "rector/rector": "^0.11.0"
```

```suggestion
        "rector/rector": "^0.11.60"
```
Let's enforce latest where your 4.3 changes are.